### DISCUSSION TITLE
Pmetrics (NP method) vs Parametric methods

### DISCUSSION THREAD

**Discussion by sumithmathew9** [LAPKB/Pmetrics#223] [General]
### Advantages of Nonparametric models over Parametric PK models

- [ ] No assumption of PK parameter distribution
- [ ] No constrain: Better detection of PK parameter variability
- [ ] Wider CV% for parameter estimates
- [ ] Better generalisability 
- [ ] Better application of Bayesian methods for MIPD (support point based), since distribution of PK parameters are captured without any artificial constrain.
- [ ] The entire support point distribution is used for concentration prediction and therefore lower risk of wrong prediction.
- [ ] Lower risk of including a wrong covariate compared to Parametric method.

### Disadvantages of Nonparametric models compared to Parametric PK models

- [ ] NP method provides only a global estimation of variance without distinction between inter-individual and inter-occasion variability (But this has not proven to affect its use in MIPD programs even in a population with a huge inter-occasion variability)
- [ ] It may be more difficult for a nonparametric model to fit an outlier patient, if the MAP estimate is outside of the support point distribution, unless we use Hybrid MAP Bayesian MM. Conversely, BestDose will be able to identify error in estimation of drug concentration (process noise) better than any other Bayesian PK software. 

Did I miss anything? 

I love to hear from those who use both Parametric and Nonparametric methods for PK modelling.

**Comment by mhovd** [comment]
I also suggest taking a look at [this paper](https://doi.org/10.1002/jcph.1993), titled _Parametric and Nonparametric Methods in Population Pharmacokinetics: Experts’ Discussion on Use, Strengths, and Limitations_.

**Reply by mhovd** [reply]
Note that there is objectively no "better" approach, it entirely depends on the problem you want to solve.

**Reply by sumithmathew9** [reply]
Thank you Markus for the paper. I would like to share another paper for those interested in a comparison between Parametric and Nonparametric models [https://pubmed.ncbi.nlm.nih.gov/32956238/] titled _Population Pharmacokinetic models of Antituberculosis Drugs in Patients: A systematic Critical review_
